src/main/java/net/openhft/chronicle/queue/reader/Reader.java

```diff
@@ -55,11 +55,10 @@ public interface Reader {
 
     /**
      * specify method reader interface to use
-     * @param methodReaderInterface interface class name. If null, a dummy reader is created
-     *                              TODO: x.23 this argument will become @NonNull
```

Comment:
should document that empty means dummy reader

Reply:
done